xjalienfs.sh

```diff
@@ -18,15 +18,14 @@ fi
 
 env PYTHONUSERBASE="$INSTALLROOT" ALIBUILD=1 python3 -m pip install --ignore-installed $PIPOPTION file://${SOURCEDIR}
 
-if [ "X$VIRTUAL_ENV" = X ]; then
-  sed -i".bak" 's/#!.*python.*/#!\/usr\/bin\/env python3/' ${INSTALLROOT}/bin/*
-  rm -v ${INSTALLROOT}/bin/*.bak
-  
-   # Uniform Python library path
-  pushd ${INSTALLROOT}
-    pushd lib
+# Make sure all the tools use the correct python
+mkdir -p ${INSTALLROOT}/bin 
+find ${INSTALLROOT}/bin -type f -exec sed -i".bak" 's/#!.*python.*/#!\/usr\/bin\/env python3/' \;
+rm -fv ${INSTALLROOT}/bin/*.bak
+
+if [ -d ${INSTALLROOT}/lib ]; then
+  pushd ${INSTALLROOT}/lib
```

Comment:
just for my understanding: Why is not enough to do consistently `#!/usr/bin/env python3` in our python scripts? @adriansev @ktf

Reply:
The problem is that when installing with pip, it uses the path of the currently used python.

Reply:
@sawenzel if you are asking why is the sed line is required for `#!/usr/bin/env python3` the reason is that the `entry_points console_scripts` mechanism of setuptools (see setup.py) place that less than optimal shebang (so outside on automatic control) .. i seen some discussions on the topic but i'm not aware of any resolution on that.